equinix/resource_network_device.go

```diff
@@ -479,6 +481,11 @@ func createNetworkDeviceSchema() map[string]*schema.Schema {
 			Computed:    true,
 			Description: neDeviceDescriptions["ZoneCode"],
 		},
+		neDeviceSchemaNames["Connectivity"]: {
+			Type:        schema.TypeString,
+			Optional:    true,
```

Comment:
Default, ForceNew, and validation function are required here. You can check isBYOL field for reference.